yt_dlp/extractor/tv5mondeplus.py

```diff
@@ -81,6 +80,14 @@ def _real_extract(self, url):
                 })
         self._sort_formats(formats)
 
+        metadata = self._parse_json(
+            vpl_data['data-metadata'], display_id)
+        duration = 0
+        if metadata:
+            content = metadata.get('content')
+            if content:
+                duration = int_or_none(content.get('duration'))
```

Comment:
```suggestion
        duration = (int_or_none(try_get(metadata, lambda x: x['content']['duration']))
                    or parse_duration(self._html_search_meta('duration', webpage)))
```
Unless u have reason to believe otherwise, assume the _html_search_meta fallback exists for a reason